src/Http/Http/src/Internal/RequestCookieCollection.cs

```diff
@@ -6,9 +6,9 @@
 using System.Collections.Generic;
 using Microsoft.Net.Http.Headers;
 
-namespace Microsoft.AspNetCore.Http.Internal
+namespace Microsoft.AspNetCore.Http
 {
-    public class RequestCookieCollection : IRequestCookieCollection
+    internal class RequestCookieCollection : IRequestCookieCollection
```

Comment:
Are we trying to get rid of internal namespaces too? Or are we fine with leaving internal types in internal namespaces?

Reply:
We can leave the types in the internal namespace and treat it as public but only if we think the usage is *really* high that it would break a large number of mainstream use-cases if we did so. We can probably get some data from apisof.net as well.

Reply:
The internal namespace seems redundant.

Reply:
I don't disagree. The only reason I think we didn't change namespaces before was to reduce the diff. I just like consistency. The good news is now we can change Kestrel's internal namespaces whenever we want.